examples/opf/experiments/classification/base_category/description.py

```diff
@@ -187,9 +187,9 @@
             # is correct here as opposed to 'columns')
             'synPermConnected': 0.1,
 
-            'synPermActiveInc': 0.1,
+            'synPermActiveInc': 0.05,
 
-            'synPermInactiveDec': 0.01,
+            'synPermInactiveDec': 0.008,
```

Comment:
did you do some testing to prove the change to the default values? 
Iff they prove better, why not change the defaults in SP/TP constructor params?